.github/workflows/pr-checks-for-qa-approvals.yml

```diff
@@ -0,0 +1,66 @@
+
+#
+# Checks if QA approvals are necessary, and if so runs the associated action when a new review is added
+#
+
+name: '[Utils] QA required approval'
+
+on:
+  pull_request:
+    types: [opened, edited, converted_to_draft, ready_for_review, reopened]
+    paths:
+      - 'VAMobile/src/**'
+      - 'VAMobile/package.json'
+      - '!VAMobile/src/**.test.*'
+  pull_request_review:
+    types: [submitted, dismissed]
+  pull_request_target:
+    branches:
+      - 'develop'
+  push: 
+    branches: 
+      - 'chanel-6261-qa-approval-workflow'     
```

Comment:
@IsraelleHub

Reply:
Please remove this trigger once you're done testing and this is ready for another round of review